src/content/defi/index.md

```diff
@@ -250,9 +260,9 @@ This means project A that received 100 donations of 1 dollar could end up with m
 
 ### Insurance {#insurance}
 
-Decentralized insurance aims to make insurance cheaper, faster to pay out, and more transparent. With more automation, cover is more affordable and pay-outs are a lot quicker. The data used to decide on your claim is completely transparent.
+Decentralized insurance aims to make insurance cheaper, faster to pay out, and more transparent. With more automation, coverage is more affordable and pay-outs are a lot quicker. The data used to decide on your claim is completely transparent.
```

Comment:
is insurance cover just a British thing?

Reply:
@ryancreatescopy I think so =)